crowdsourcing/models.py

```diff
@@ -467,4 +468,24 @@ class ConversationRecipient(models.Model):
 class UserMessage(models.Model):
     message = models.ForeignKey(Message)
     user = models.ForeignKey(User)
-    deleted = models.BooleanField(default=False)
\ No newline at end of file
+    deleted = models.BooleanField(default=False)
+
+class File(models.Model):
+    #TODO will need save files on a server rather than in a temporary folder
+    file = models.FileField(upload_to='tmp/')
+    deleted = models.BooleanField(default=False)
+
+    def parse_csv(self):
```

Comment:
as this is just processing the file content i.e it's representation it is supposed to be here, and it will be reused by the other backend classes, this is all good